### PR TITLE
cmus: add libao dependency

### PR DESCRIPTION
**Pull request by lingawakad**
On MacOS Monterey and Ventura, cmus with Core Audio output plugin produces audio and output switching issues. Re-adding libao as an output plugin seems to solve these issues.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Is there an upstream issue you can reference here?

**Comment by lingawakad** [comment]
> Is there an upstream issue you can reference here?

Here's an issue that's still open, which has the recommendation - https://github.com/cmus/cmus/issues/1130

**Comment by SMillerDev** [comment]
Seems from that issue this only applies to arm though?

**Comment by lingawakad** [comment]
I'm on an Intel model and it seems to have solved my issues. Is there a different repo it should go into for ARM?

**Comment by SMillerDev** [comment]
I think that's important information to add to that issue then. It's a shame upstream isn't looking into the cause of the issue though.

**Comment by lingawakad** [comment]
> I think that's important information to add to that issue then. It's a shame upstream isn't looking into the cause of the issue though.

Added a comment on the upstream issue. Thanks for the suggestion!

File: Formula/cmus.rb [hunk 1]
```diff
@@ -33,15 +34,20 @@ class Cmus < Formula
...
-
   def install
-    system "./configure", "prefix=#{prefix}", "mandir=#{man}",
-                          "CONFIG_WAVPACK=n", "CONFIG_MPC=n"
+    args = [
+      "prefix=#{prefix}",
+      "mandir=#{man}",
+      "CONFIG_WAVPACK=n",
+      "CONFIG_MPC=n",
+      "CONFIG_AO=y"
```
Comment (SMillerDev) [inline Formula/cmus.rb:43]:
```suggestion
      "CONFIG_AO=y",
```

**Comment by SMillerDev** [comment]
Please don't add merge commits, our CI can't auto-merge with those.

**Comment by lingawakad** [comment]
> .

Whoops, sorry. Need me to redo the PR?

**Comment by SMillerDev** [comment]
You can rebase and drop the merge commit

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**